ui/src/app/shared/header/header.component.ts

```diff
@@ -115,13 +115,16 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy {
         let keyVal: string[] = value.split(':');
         this.deleteChipIfExists(keyVal[0].trim());
         this.chips.set(keyVal[0].trim(), keyVal[1].trim());
+        this.search();
       }
       else {
         // Parse as just the key
         this.deleteChipIfExists(value);
         this.chips.set(value.trim(), '');
```

Comment:
Would it be easy to trigger showing the equals/date drop down menu from this? It still feels odd to add a blank chip

Reply:
Agreed that this should be the behavior, and I plan that this will be a part of the effort to open/close menus and auto focus where needed. Still figuring out the best way to do this, since it will likely involve tightly coupling the DOM to the app logic which generally degrades code quality.